File: snippets/java/delete_field_index.java
import io.qdrant.client.QdrantClient;
import io.qdrant.client.QdrantGrpcClient;

QdrantClient client = new QdrantClient(
                QdrantGrpcClient.newBuilder("localhost", 6334, false).build());

client.deletePayloadIndexAsync("{collection_name}", "{field_name}", true, null, null).get();

File: snippets/go/delete_field_index.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func deleteFieldIndex() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	_, err = client.DeleteFieldIndex(context.Background(), &qdrant.DeleteFieldIndexCollection{
		CollectionName: "{collection_name}",
		FieldName:      "{field_name}",
	})
	if err != nil {
		panic(err)
	}
}
